packages/dds/legacy-dds/src/test/array/fuzzUtils.ts

```diff
@@ -109,6 +138,19 @@ export function makeSharedArrayReducer<T extends SerializableTypeForSharedArray>
 			client.channel.delete(index);
 		},
 		move: ({ client }, { oldIndex, newIndex }) => {
+			const array = client.channel;
+			if (!moveListenerRegistry.has(array.id)) {
+				moveListenerRegistry.add(array.id);
+				// Register listener to track move entry IDs
+				array.on("valueChanged", (op, isLocal, _target) => {
+					if (op.type === OperationType.moveEntry && isLocal) {
```

Comment:
I don't have context on the SharedArray API. Is it actually a constraint with toggleMove that the corresponding id has to be local? If so, this strategy looks good. Otherwise I think you probably want to use the `onClientCreate` event emitted by the harness to attach an event listener that populates/updates this array.